R/ESA.R

```diff
@@ -0,0 +1,101 @@
+# ESA.R
+#
+# Purpose: Exploratory System Analysis tool project for BCB420 2019
+# Version: 1.0.0
+# Version history: 1.0.0
+# Date: 25 March 2019
+# Author: Deus Bajaj (deus.bajaj@mail.utoronto.ca)
+# License: MIT
+#
+# Input: The biological system PHALY and a gene the user picks from the system
+# Output: A histogram for the node degree distribution of the PHALY gene network
+# and a network plot for a gene from the PHALY system
+# Dependencies: devtools, iGraph
+#
+
+# ====  FUNCTIONS  =============================================================
+
+# function stub taken from Dr. Steipe's BCB420.2019.ESA package (https://github.com/hyginn/BCB420.2019.ESA)
+fetchComponents <- function(sys) {
```

Comment:
You don't need this function stub, and fetchComponents has been deprecated anyways. Please use `SyDBgetSysSymbols(myDB, sys)` instead